### PR TITLE
Casey fd merge

### PR DESCRIPTION
**Pull request by frnkdny**

### CODE REVIEW

File: gce/Makefile [hunk 1]
```diff
@@ -135,7 +135,8 @@ get-results:
 
 # See http://stackoverflow.com/a/12110773/61318
 #make -j12 get-diags
-get-diags: describe-nodes describe-pods describe-services describe-rc describe-ep get-prom-data $(LOG_RETRIEVAL_TARGETS) get-pod-logs
+get-diags: describe-nodes describe-pods describe-services describe-rc describe-ep get-prom-data $(LOG_RETRIEVAL_TARGETS)
```
Comment (caseydavenport) [inline gce/Makefile]: Why did you remove get-pod-logs from this?

File: gce/policy-scale-test/get-data.py [hunk 2]
```diff
@@ -52,17 +52,17 @@ def collect_data():
                              "-o", "json"])
     all_pods = json.loads(all_pods)["items"]
 
-    # Get calico-k8s-policy-agent pod metadata if it is running.
+    # Get calico-policy-controller (agent) pod metadata if it is running.
```
Comment (caseydavenport) [inline gce/policy-scale-test/get-data.py]: No need for `(agent)` here, but fine to leave as is.

File: gce/policy-scale-test/get-data.py [hunk 3]
```diff
@@ -222,9 +222,13 @@ def display_data():
 
     # Print out some data.
     startup_time = (max_x - min_x).seconds
-    print "Time to start %s pods: %s (%s pods/s)" % (len(x),
-                                                     startup_time,
-                                                     len(x)/startup_time)
+    if startup_time != 0:
```
Comment (caseydavenport) [inline gce/policy-scale-test/get-data.py]: When do we expect startup time to be 0?

**Comment by caseydavenport** [comment]
@frnkdny stuck some comments on the latest commit.

File: gce/policy-scale-test/get-data.py (hunk 3 above)
Comment (frnkdny) [inline gce/policy-scale-test/get-data.py]: When running this at small scale 2nodes and 4pods, I was consistently hitting pod start times within the same second. This was giving me a ZeroDivisionError since max_x and min_x would be the same value.
